src/explore-education-statistics-frontend/src/modules/table-tool/components/TableTool.tsx

```diff
@@ -40,6 +44,18 @@ const TableToolFinalStep = ({
     setPermalinkId('');
   }, [tableHeaders, table]);
 
+  useEffect(() => {
+    const setContentPublicationData = async () => {
+      if (publication) {
+        const response = await publicationService.getLatestPublicationRelease(
```

Comment:
Getting the full (and pretty heavy weight) release model here seems excessive.

I would suggest adding a new api endpoint that returns just the methodology details

Reply:
An alternative to making a new endpoint would be to specifiy what fields we actually need i.e. a `fields` query parameter.

Making a new endpoint is simpler, but would most likely need some URL `/methology/publication/{publicationSlug}` which isn't particularly RESTful (there's plenty of examples of us violating this though, so whatever I guess).

Reply:
Done